### ISSUE TITLE
String length inconsistency

### ISSUE THREAD

**Issue by JefHellemans** [airbnb/javascript#1043]
In [6.2](https://github.com/airbnb/javascript#strings--line-length) it is stated that long strings shouldn't be broken up over multiple lines using concatenation.
```
// bad
const errorMessage = 'This is a super long error that was thrown because \
of Batman. When you stop to think about how Batman had anything to do \
with this, you would get nowhere \
fast.';

// bad
const errorMessage = 'This is a super long error that was thrown because ' +
  'of Batman. When you stop to think about how Batman had anything to do ' +
  'with this, you would get nowhere fast.';

// good
const errorMessage = 'This is a super long error that was thrown because of Batman. When you stop to think about how Batman had anything to do with this, you would get nowhere fast.';
```

However, later on in [8.3 and 8.4](https://github.com/airbnb/javascript#arrows--paren-wrap) concatenation is encouraged.
```
// bad
[1, 2, 3].map(number => 'As time went by, the string containing the ' +
  `${number} became much longer. So we needed to break it over multiple ` +
  'lines.'
);

// good
[1, 2, 3].map(number => (
  `As time went by, the string containing the ${number} became much ` +
  'longer. So we needed to break it over multiple lines.'
));
```
```
// good
[1, 2, 3].map(number => (
  `A long string with the ${number}. It’s so long that we’ve broken it ` +
  'over multiple lines!'
));
```

**Comment by vgvenkat** [comment]
I can submit a pr if idea is to go with 6.2 string format. I guess this is just modifying 8.3 and 8.4 and nothing more.

**Comment by ljharb** [comment]
@vgvenkat exactly right - that'd be appreciated!

**Comment by vgvenkat** [comment]
I guess these were fixed in https://github.com/airbnb/javascript/pull/1065 more than a month ago. We can close this issue.

**Comment by ljharb** [comment]
Whoops, thanks - fixed in #1065.
